infection.json.dist

```diff
@@ -4,6 +4,7 @@
             "src"
         ]
     },
+    "timeout": 30,
```

Comment:
Add this to `yiisoft/package-template` if it need

Reply:
This is specific to packages where the tests take a little longer, in this case, because of the access to the database.